web-console/future-openshift/diagnostics/diagnostics.md

```diff
@@ -0,0 +1,37 @@
+# Diagnostic Report
+
+Users can download a diagnostic report of their system and send it to Red Hat from the console.
+
+
+## Help menu
+![dropdown](img/dropdown.png)
+* The "Diagnostic Report" option is available from the help menu.
+
+## Modal
+![modal](img/modal.png)
+* A modal will open with a description of how the diagnostic report will be used.
+* Users can indicate what they would like to include in the diagnostic report
+  * If users choose to include node health, a multiselect dropdown will allow them to select which ones.
+* Users can indicate whether or not they have an open support case already.
+  * If users have a case number they can enter it into the "Case number" field.
+  * If users do not have an open case they may choose to open one. If they choose this option, a link to the case will be included in the success screen.
+  * Users may also choose to just download the diagnostic report to their machine and send it to Red Hat at a later time.
+
+## In progress
+![generating](img/generating.png)
+* Diagnostic reports may take a while to download. A progress bar is provided for users to track the download progress. Users may also close the modal during the download.
+* The "Download" button is disabled to prevent users from generating a report multiple times.
+
+## Success
```

Comment:
Should the primary action be "Generate" instead of Download if the user still needs to grab the report itself from a URL or location?

Reply:
Is it also showing the progress of downloading (in addition to generating) in the progress bar or would the download to your computer automatically initiate after this in 100% generated? I would assume either way this would download automatically based on the action button reading "download" where you initiate the diagnostics – if not, I agree with changing the name to "generate" (but if users always want to download it after generating it would be ideal to initiate downloading without requiring an additional action)